README.md

```diff
@@ -227,6 +230,10 @@ To run the code in this repository, you must have Node.js 10+ on your computer a
 
 Go to the [SDK Documentation](https://lightclient.raiden.network/docs/) for more information on how to install and use the SDK.
 
+### CLI Documentation
```

Comment:
```suggestion
### CLI/API Documentation
```
Check if the reference in the table of contents still works with my change. Thanks :)